bundles/specmate-search/src/com/specmate/search/internal/services/LuceneBasedModelSearchService.java

```diff
@@ -124,8 +124,7 @@
 	private AtomicBoolean isReindexRunning = new AtomicBoolean(false);
 
 	/** List of classes included in the index */
-	private List<String> indexedClasses = Arrays.asList("Requirement", "CEGModel", "TestSpecification",
-			"TestProcedure");
+	private List<String> indexedClasses = Arrays.asList("Requirement", "CEGModel", "TestSpecification", "Process");

```

Comment:
I wonder if it makes sense to make this configurable. Maybe with the current set as a default. I remember discussions whether also to include test procedures. @wehrlean what do you think?

Reply:
I am not sure if it is really needed by a user. It could lead to over-configuration. But I have no insights, how Specmate is used in reality...

Reply:
We said to exclude the test-procedures because you normally get there via the testcases